Tools/CISupport/ews-build/steps.py

```diff
@@ -5004,16 +5004,17 @@ def get_change_status(self, change_id, queue):
         try:
             response = yield TwistedAdditions.request(url, logger=lambda content: self._addToLog('stdio', content))
             if response.status_code != 200:
```

Comment:
but why was response.status_code non-200 in the first place?

Reply:
That's not the case we're hitting, we're hitting the `Exception`